### ISSUE TITLE
MongoDataProvider fails with MongoDB.Driver v3.0.0

### ISSUE THREAD

**Issue by emeryao** [thepirat000/Audit.NET#708]
**Describe the bug**
An exception occured when inserting a new audit event with logs below
```
System.TypeLoadException: Could not load type 'MongoDB.Driver.MongoClientBase' from assembly 'MongoDB.Driver, Version=3.0.0.0, Culture=neutral'.
   at Audit.MongoDB.Providers.MongoDataProvider.GetDatabase()
   at Audit.MongoDB.Providers.MongoDataProvider.InsertEventAsync(AuditEvent auditEvent, CancellationToken cancellationToken)
   at Audit.Core.AuditScope.SaveEventAsync(Boolean forceInsert, CancellationToken cancellationToken)
```

**To Reproduce**
Create an audit event and try to insert to mongodb with MongoDataProvider

**Expected behavior**
MongoDataProvider should work with MongoDB.Driver 3.0.0

**Libraries (specify the Audit.NET extensions being used including version):**
- Audit.NET.MongoDB: 27.1.0
 - Audit.WebApi.Core: 27.1.0
 - [MongoDB.Driver: 3.0.0](https://github.com//mongodb/mongo-csharp-driver/releases/tag/v3.0.0)

**Target .NET framework:**
.NET SDK: 8.0.403

---
Sorry for my poor English

**Comment by thepirat000** [comment]
This issue was resolved in version 27.1.1. Please update your references and re-test.

As of version 27.1.1, the Audit.NET.MongoDB library now uses the latest MongoDB driver, v3.0.0

**Comment by emeryao** [comment]
Thanks for the quick response

version 27.1.1 works perfectly with MongoDB driver v3.0.0 for my project now

**Comment by syedalinaqi** [comment]
@thepirat000 this still seems to appears for AddAuditSubscriber

**Comment by thepirat000** [comment]
This was fixed in version 27.3.0, please upgrade your references and re-test
